### ISSUE TITLE
Improve flail functionality

### ISSUE THREAD

**Issue by Sunconure11** [ThexXTURBOXx/balkons-weaponmod#39]
Just what the title says. Flails already act... weird.

**Comment by ThexXTURBOXx** [comment]
Indeed, I had also noticed that already... Probably I will do this after some general stability and bug fixes

**Comment by Sunconure11** [comment]
@ThexXTURBOXx When do you plan on releasing a build of Expanded Armory? Also, do you have a Discord at all? It may help with relaying stuff better, such as finding mods that could benefit.

**Comment by ThexXTURBOXx** [comment]
Good question. I have quite a large backlog of things I want to accomplish. First of all, I want to finish a few scientific reviews for my career as a researcher - after that I want to finish the next update for [Block Helper](https://github.com/ThexXTURBOXx/BlockHelper) and thenI will focus more on BWM: Legacy again. I am planning on releasing a build together with the next update for BWM: Legacy - no idea when, though!
